File: problems/problems_3356/solution.go
package problem3356

import (
	"encoding/json"
	"log"
	"strings"
)

func minZeroArray(nums []int, queries [][]int) int {
    
}

func Solve(inputJsonValues string) any {
	inputValues := strings.Split(inputJsonValues, "\n")
	var nums []int
	var queries [][]int

	if err := json.Unmarshal([]byte(inputValues[0]), &nums); err != nil {
		log.Fatal(err)
	}
	if err := json.Unmarshal([]byte(inputValues[1]), &queries); err != nil {
		log.Fatal(err)
	}

	return minZeroArray(nums, queries)
}

File: problems/problems_3356/solution.ts
function minZeroArray(nums: number[], queries: number[][]): number {
    
};

export function Solve(inputJsonElement: string): any {
	const inputValues: string[] = inputJsonElement.split("\n");
	const nums: number[] = JSON.parse(inputValues[0]);
	const queries: number[][] = JSON.parse(inputValues[1]);
	return minZeroArray(nums, queries);
}
